src/App.js

```diff
@@ -31,37 +35,35 @@ export default function App() {
   const history = useHistory()
 
   useEffect(() => {
-    const historicCarbonFootprint = JSON.parse(
-      localStorage.getItem('Carbon Footprint History')
+    const historicCarbonFootprint = getFromStorage(
+      APP_STORAGE_KEYS.footprintHistory
     )
+
     historicCarbonFootprint && setCarbonFootprint(historicCarbonFootprint)
-    const historicTotalCarbonFootprint = JSON.parse(
-      localStorage.getItem('Total Carbon Footprint')
+
+    const historicTotalCarbonFootprint = getFromStorage(
+      APP_STORAGE_KEYS.footprintTotal
     )
-    const historyFootprintPerTransportationType = JSON.parse(
-      localStorage.getItem('Footprint per Transportation Type')
+
+    const historyFootprintPerTransportationType = getFromStorage(
+      APP_STORAGE_KEYS.footprintPerTransportType
     )
+
     setTotalCarbonFootprint(historicTotalCarbonFootprint)
     setFootprintPerTransportationType(
       historyFootprintPerTransportationType || []
     )
   }, [])
 
   useEffect(() => {
-    localStorage.setItem(
-      'Carbon Footprint History',
-      JSON.stringify(carbonFootprint)
-    )
+    saveToStorage(APP_STORAGE_KEYS.footprintHistory, carbonFootprint)
     setTotalCarbonFootprint(calculateTotalFootprintSum(carbonFootprint))
 
-    localStorage.setItem(
-      'Total Carbon Footprint',
-      JSON.stringify(totalCarbonFootprint)
-    )
+    saveToStorage(APP_STORAGE_KEYS.footprintTotal, totalCarbonFootprint)
 
-    localStorage.setItem(
-      'Footprint per Transportation Type',
-      JSON.stringify(footprintPerTransportationType)
+    saveToStorage(
```

Comment:
Hier findet die mehrzeilige Schreibweise gut, allerdings verwirrt mich persönlich der Einzeiler in Zeile 54.

Reply:
Hm, das hat Prettier automatisch gemacht. Da schau ich mal rein 👍